models/example/order_calculations.sql

```diff
@@ -0,0 +1,30 @@
+with source1 as (
+
+    select * from {{ ref('customers_upload') }}
```

Comment:
so you don't want to ref from the `customers_upload` but instead reference from your `stg_customers` model. you want to limit the number of times you pull from the raw data to exactly once so that if you rename, change things you only do it in one location